include/admin_navigation.js

```diff
@@ -257,14 +305,26 @@ module.exports = function AdminNavigationModule(pb) {
      * @method buildNavigation
      * @returns {Array}
      */
-    function buildNavigation() {
+    function buildNavigation(site) {
         var i;
         var navigation = [];
-        var defaultNavigation = getDefaultNavigation();
-        var additions = getAdditions();
-        var childrenAdditions = getChildrenAdditions();
+        var multiSiteAdditions = getMultiSiteNavigation();
+        var defaultNavigation = getDefaultNavigation(site);
```

Comment:
These have been moved accordingly.

Ben Tidwell
Software Engineer III | Talent Network
CareerBuilder
5550-A Peachtree Pkwy, Norcross GA 30092

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 9, 2015 at 5:56 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In include/admin_navigation.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34308622>:

>          var i;
>          var navigation = [];
> -        var defaultNavigation = getDefaultNavigation();
> -        var additions = getAdditions();
> -        var childrenAdditions = getChildrenAdditions();
> +        var multiSiteAdditions = getMultiSiteNavigation();
> +        var defaultNavigation = getDefaultNavigation(site);


The calls for multi-stie navigation and default navigation don't always have to be made. Only make the function call inside of the if/else block that they pertain too.

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34308622>.